packages/graphql-server/src/resolvers/configurations/configurations.d.ts

```diff
@@ -14,6 +14,7 @@ export type ConfigurationType =
   | '​​tenancyTypes'
   | '​​vendorTypes'
   | '​​worksOrderTypes'
+  | 'keyTypes'
```

Comment:
So I think because keys is part of properties, it should be an extension of the properties API itself. The aim has always been to be a 1:1 map of the main API schema so feel this might be going off piste!